bugs/gh-575.txt

```diff
@@ -0,0 +1,246 @@
+Launched instance: LXDInstance(name=caring-crayfish)
+.Detected image: image_id=focal os=ubuntu release=focal
+Launching instance with launch_kwargs:
+user_data=runcmd:
+  - echo 'hi' > /var/tmp/test
+
+image_id=local:vital-vulture-snapshot
+wait=False
+['lxc', 'init', 'local:vital-vulture-snapshot', '--config', "user.user-data=runcmd:\n  - echo 'hi' > /var/tmp/test\n"]
+Launched instance: LXDInstance(name=one-teal)
+.Deleting snapshot image created for this testrun: local:vital-vulture-snapshot
+
+
+======================== 2 passed in 128.40s (0:02:08) =========================
+
+=== Groovy ===
+============================= test session starts ==============================
+platform linux -- Python 3.5.9, pytest-5.4.2, py-1.8.1, pluggy-0.13.1
+rootdir: /home/james/cloud-init, inifile: tox.ini
+plugins: cov-2.9.0
+collected 2 items
+
+tests/integration_tests/modules/test_cli.py Detected image: image_id=groovy os=ubuntu release=groovy
+Detected image: image_id=groovy os=ubuntu release=groovy
+Settings:
+CLOUD_INIT_SOURCE=PROPOSED
+COLLECT_LOGS=NEVER
+EXISTING_INSTANCE_ID=None
+GCE_PROJECT=ubuntu-server-277015
+GCE_REGION=us-central1
+GCE_ZONE=a
+INSTANCE_TYPE=None
+KEEP_IMAGE=False
+KEEP_INSTANCE=False
+LOCAL_LOG_PATH=/tmp/cloud_init_test_logs
+OCI_COMPARTMENT_ID=ocid1.compartment.oc1..aaaaaaaanz4b63fdemmuag77dg2pi22xfyhrpq46hcgdd3dozkvqfzwwjwxa
+OS_IMAGE=groovy
+PLATFORM=lxd_container
+RUN_UNSTABLE=False
+Setting up environment for lxd_container
+Launching instance with launch_kwargs:
+user_data=None
+wait=False
+image_id=ubuntu:bac1692e9ec7f5c5cee35d149ca789121bed238c1182df2134b5aefa3977f3e0
+['lxc', 'init', 'ubuntu:bac1692e9ec7f5c5cee35d149ca789121bed238c1182df2134b5aefa3977f3e0']
+Launched instance: LXDInstance(name=mature-rhino)
+Installing proposed image
+Installed cloud-init version: 20.4-0ubuntu1~20.10.1
+Created new image: local:mature-rhino-snapshot
+Done with environment setup
+Launching instance with launch_kwargs:
+user_data=#cloud-config
+runcmd:
+  - echo 'hi' > /var/tmp/test
+
+wait=False
+image_id=local:mature-rhino-snapshot
+['lxc', 'init', 'local:mature-rhino-snapshot', '--config', "user.user-data=#cloud-config\nruncmd:\n  - echo 'hi' > /var/tmp/test\n"]
+Launched instance: LXDInstance(name=guiding-sole)
+.Detected image: image_id=groovy os=ubuntu release=groovy
+Launching instance with launch_kwargs:
+user_data=runcmd:
+  - echo 'hi' > /var/tmp/test
+
+wait=False
+image_id=local:mature-rhino-snapshot
+['lxc', 'init', 'local:mature-rhino-snapshot', '--config', "user.user-data=runcmd:\n  - echo 'hi' > /var/tmp/test\n"]
+Launched instance: LXDInstance(name=present-tiger)
+.Deleting snapshot image created for this testrun: local:mature-rhino-snapshot
+
+
+======================== 2 passed in 126.29s (0:02:06) =========================
+
+[Regression Potential]
+None. It's a new command that doesn't run unless invoked.
```

Comment:
This isn't a new command, it's a new switch to an existing command. We could potentially have regressed existing users of the `schema` subcommand if the modifications have introduced a bug that we haven't caught.